R/test_wrapper.R

```diff
@@ -1761,19 +1863,19 @@ tidy.anova_exploratory <- function(x, type="model", conf_level=0.95, levene_test
     orig_terms[is.na(orig_terms)] <- colnames(ret)[is.na(orig_terms)] # Fill the column names that did not have a matching mapping.
     colnames(ret) <- orig_terms
   }
-  else if (type == "prob_dist") {
+  else if (type == "prob_dist") { # Note that 2-way ANOVA, with multiple P-values in the result, is not supported here.
```

Comment:
Update comment.

Reply:
Done (Removed the comment that does not hold true anymore.)